app/views/reductions.html

```diff
@@ -101,7 +101,7 @@ <h2 id="headingScheduled" class="govuk-heading-l">Scheduled<span class="visually
 
     <h2 id="headingArchived" class="govuk-heading-l">Archived<span class="visuallyhidden"> reductions</span></h2>
 
-    <table class="govuk-table sln-table-reductions" id= "override-govuk-table">
```

Comment:
there is unfortunately some styling hanging off this id which we will need to check - see https://github.com/ministryofjustice/wmt-web/blob/master/app/assets/sass/application.scss#L190